src/content/blog/astro-490.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Astro 4.9"
+description: "Astro 4.8 is now available! This release includes the long-awaited Container API, stabilized experimental features, and more."
```

Comment:
```suggestion
description: "Astro 4.9 is now available! This release includes the long-awaited Container API, stabilized experimental features, and more."
```
Sharing a preview of the post with Joe and seeing the description in the preview card, FTW!  😅